invenio_app_rdm/records_ui/searchapp.py

```diff
@@ -0,0 +1,230 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2018-2021 CERN.
+#
+# Invenio-App-RDM is free software; you can redistribute it and/or modify
+# it under the terms of the MIT License; see LICENSE file for more details.
+
+"""Configuration helper for React-SearchKit."""
+
+from copy import deepcopy
+from functools import partial
+
+from flask import current_app
+from invenio_rdm_records.searchconfig import OptionsSelector, \
+    SortOptionsSelector
+
+
+class SortConfig(SortOptionsSelector):
+    """Sort options for the search configuration."""
+
+    def map_option(self, key, option):
+        """Generate a RSK search option."""
+        return {
+            "sortBy": key,
+            "text": option['title']
+        }
+
+
+class FacetsConfig(OptionsSelector):
+    """Facets options for the search configuration."""
+
+    def map_option(self, key, option):
+        """Generate an RSK aggregation option."""
+        title = option.get('title', option['facet']._label)
+
+        ui = deepcopy(option['ui'])
+        ui.update({
+            'aggName': key,
+            'title': title,
+        })
+
+        # Nested facets
+        if 'childAgg' in ui:
+            ui['childAgg'].set_default('aggName', 'inner')
+            ui['childAgg'].set_default('title', title)
+
+        return ui
+
+
+class SearchAppConfig:
+    """Configuration generator for React-SearchKit."""
+
+    default_options = dict(
+        endpoint=None,
+        hidden_params=None,
+        app_id='rdm-search',
+        headers=None,
+        list_view=True,
+        grid_view=True,
```

Comment:
We had `grid_view=False` before. I think because the UI/frontend code for grid isn't quite there yet.